src/v2/services/address.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const addressValidator = require('../modules/addresses/validator');
+const addressRepo = require('../connectors/repository/addresses');
+
+const createAddress = async address => {
+  const { error, value: validatedAddress } = addressValidator.validate(address);
+
+  if (error) {
```

Comment:
I've started a pattern both in the CRM and water service, where there is a custom error type that is thrown within the service layer code, and then the controller maps those errors to Boom errors at the controller layer.
I think we should stick with that here and throw a BadEntityError which can map in the controller to a 422.

Reply:
OK. I can change to use that pattern. I don't really like the idea of throwing an error because if it is not handled it will terminate (and for use restart) the service. 

However it will make the code all look the same.